### ISSUE TITLE
collections.template_err_save

### ISSUE THREAD

**Issue by mdehaan** [modxcms/Collections#124]
When I save a template then it doesn't work and get error (collections.template_err_save).
MODx 2.3.1-pl
Collections-3.1.1-pl

**Comment by TheBoxer** [comment]
Can you provide more info? I have no problem saving Collection views...

**Comment by mdehaan** [comment]
On mij existing MODX install I tried to uninstall/install a coumple of times, but that didn't solve the problem.
I manually deleted the collection database tables and reinstalled, and after that it worked fine.

Thanx for developing this GREAT plugin TheBoxer.
This was the one thing missing from the MODX interface.

**Comment by TheBoxer** [comment]
Interesting that it didn't work... I tried to upgrading from 3.0.0 on local and it was fine...
probably some weird bug :)

you're welcome ;)

**Comment by jonleverrier** [comment]
Just as a quick note - i've run into the same problem. Not sure what is triggering this problem...

* I was running MODX 2.4.0, with the last major release of collections downloaded but not installed (it showed in my packages grid)
* I upgraded MODX to 2.4.1 and upgraded collection to 3.3.0.
* I assigned Collections to a resource, which returned an empty grid. 
* I went to Collections views, tried to setup a new view, hit save and received "collections.template_err_save"
